dockerfiles/mingw-fedora/Dockerfile

```diff
@@ -0,0 +1,55 @@
+FROM fedora:30
+
+# Update base system
+RUN dnf update -y
+
+# Add some useful packages to the base system
+RUN dnf install -y \
+        make \
+        cmake \
+        git \
+        sudo \
+        wget \
+        gawk \
+        sudo \
+        fakeroot \
+        file \
+        patch \
+        ninja-build
+
+# Install MingW packages
+RUN dnf install -y \
+        mingw32-binutils \
+        mingw32-boost \
+#        mingw32-cmake \
```

Comment:
This looks off. I doubt you can insert comments in the middle of a list using escaped new lines.

Reply:
You can comment in the middle of escaped new lines in Dockerfiles. I was trying to copy the list from the mingw-arch Dockerfile. It's commented out because there is no mingw cmake package on fedora. I had it commented out to get the Dockerfile to build, but it can be removed now as it probably doesn't need to be there.